photon-core/src/main/java/org/photonvision/common/hardware/HardwareManager.java

```diff
@@ -165,6 +165,53 @@ public boolean restartDevice() {
         }
     }
 
+    /**
+     * Get the MAC address of the device
```

Comment:
More specifically, this looks like the MAC address of the primary NIC in use. 

If we are looking for a hardware UUID for a particular SBC, we could consider something like this right?

https://stackoverflow.com/a/10152797